packages/reakit/src/Step/Step.tsx

```diff
@@ -14,6 +14,7 @@ export interface StepProps extends HiddenProps {
   update: StepContainerActions["update"];
   unregister: StepContainerActions["unregister"];
   order?: number;
+  active?: boolean;
```

Comment:
Maybe we should just drop this prop. I don't think we're using it. 🤔
Nor this is documented.

Reply:
Ok, i prefered to keep it instead of breaking everything.